min_compute.yml

```diff
@@ -71,7 +71,7 @@ compute_spec:
       ram_type: "DDR4"     # RAM type (e.g., DDR4, DDR3, etc.)
 
     storage:
-      min_space: 24           # Minimum free storage space (GB)
+      min_space: 40           # Minimum free storage space (GB)
```

Comment:
I'm not entirely sure if 40 is the minimum space to run a validator with llama3, but I believe you have more perspective on this than I do. Worst case scenario we might revisit this doc in a near future  ;)

Reply:
This model weights ~37GB. We can make it larger to be safe

Reply:
> I'm not entirely sure if 40 is the minimum space to run a validator with llama3, but I believe you have more perspective on this than I do. Worst case scenario we might revisit this doc in a near future ;)

This model weights ~37GB. We can increase it a little to be safe